src/cpp/CMakeLists.txt

```diff
@@ -604,6 +604,12 @@ elseif(RSTUDIO_CONFIG_SESSION_ONLY)
    message(STATUS, "Configured to build SESSION_ONLY")
 endif()
 
+# the core libs can be built without libpam (for Launcher, originally), but we
+# use it by default
+if (NOT DEFINED RSTUDIO_USE_PAM)
+  message(STATUS "Configured to build with libpam features")
```

Comment:
Minor nit: this check should happen outside of the `NOT DEFINED` block here just so we get a status report when one explicitly sets `cmake -DRSTUDIO_USE_PAM`.